styles.tsx

```diff
@@ -0,0 +1,35 @@
+import { StyleSheet } from "react-native";
+
+const style = StyleSheet.create({
+  container: {
+    backgroundColor: "#FFF7E6",
+    paddingVertical: 50,
+    paddingHorizontal: 20,
+    justifyContent: "center",
+  },
+  buttonStyle: {
+    // TODO: Add generic button style for touchable opacity component
+    backgroundColor: "#FFC1C1",
+    marginTop: 20,
+    margin: 40,
+    padding: 20,
+    borderRadius: 40,
+  },
+  butonTextStyle: {
+    //TODO
+    color: "#4A4A4A", // dark grey
+    textAlign: "center",
```

Comment:
I would suggest you to creat a `Theme.tsx` file in the `types/` folder that holds a bunch of colour and other property values. This way you can reuse them throughout the project and maintain theme consistency. Example:

```js
Color {
    const DARK_GRAY = "#4a4a4a"
}
```
This way, you can directly use `Color.DARK_GRAY` wherever needed.

Decide which data structure is best for this approach. Either named enums or objects or something else.

Reply:
I have created a file `Theme.tsx'  to store colors as directed and I have used Objects for this. Check once pls

Reply:
It's alright for now. Although I was thinking of a more versatile approach. Maybe when the need arises, you should be ready to refactor.